urx/urrtmon.py

```diff
@@ -254,7 +283,7 @@ def __recv_rt_data(self):
             self._ctrlTimestamp = np.array(unp[0])
             if self._last_ctrl_ts != 0 and (
                     self._ctrlTimestamp -
-                    self._last_ctrl_ts) > 0.010:
+                    self._last_ctrl_ts) > 0.1:
```

Comment:
same here. The robot sends data at 100Hz, the timestamp should be accordingly, if no packet is send in over 0.1s there is something bad happenign and we need to discover it